extensions/extensions-docker/pom.xml

```diff
@@ -42,32 +41,6 @@
             <properties>
                 <dockerImageVersion>${project.version}-${target.architecture}</dockerImageVersion>
             </properties>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.codehaus.mojo</groupId>
-                        <artifactId>exec-maven-plugin</artifactId>
```

Comment:
I: This was removed while debugging the GH build action.  To try and save disk space on the community runner we were wiping the buildkit layer cache on each docker build.  This is no longer needed with self-hosted runners and was causing a lot of churn for the release process (and builds in general).